users/models.py

```diff
@@ -3,10 +3,13 @@
 
 
 def upload_location(instance, filename):
-	return "{}/{}".format(instance.id, filename)
+    return "{}/{}".format(instance.id, filename)
 
 class Profile(models.Model):
-	user = models.OneToOneField(User, on_delete=models.CASCADE)
-	initials = models.CharField(max_length=8)
-	bio = models.CharField(max_length=200)
-	
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+    initials = models.CharField(max_length=8)
```

Comment:
Remove. just add a method to get the initials of the user based on their first and last name